tests/test_equilibrium.py

```diff
@@ -464,3 +465,143 @@ def test_backward_compatible_load_and_resolve():
     f_obj = ForceBalance(eq=eq)
     obj = ObjectiveFunction(f_obj)
     eq.solve(maxiter=1, objective=obj)
+
+
+@pytest.mark.unit
+def test_shifted_circle_geometry():
+    """
+    In this test, we calculate a low-beta shifted circle equilibrium with DESC.
+
+    We then compare the various geometric coefficients with their respective analytical
+    expressions.
+    """
+    eq = Equilibrium.load(".//tests//inputs//low-beta-shifted-circle.h5")
```

Comment:
looks like this file isn't included in the PR

Reply:
Done.